app/Console/Commands/EditImagesCommand.php

```diff
@@ -45,12 +46,17 @@ public function handle(AWS $aws)
         Post::where('id', '>', $start)->chunk(100, function ($posts) use ($aws) {
             foreach ($posts as $post) {
                 $image = $post->url;
```

Comment:
Great catch! I'm wonder if since we know that some of the images are `default` if in addition to the other things you've added if after this line we should have
`if (`$image` === "default") {
    return;
 }`
so that then we are printing out errors when we got a true error? Just a thought, what you have is totally functional!